README.md

```diff
@@ -41,7 +47,8 @@ class Album(models.Model):
     artist = models.ForeignKey(Artist, on_delete=models.CASCADE)
     cover = models.CharField(max_length=256, null=True, default=None)
 ```
-And you would like to filter results in Album Admin on the basis of artist, then you can define `search fields` in Artist and then define filter as:
+
+And you would like to filter results in `AlbumAdmin` on the basis of `artist`.  You need to define `search fields` in `Artist` and then define filter like this:
```

Comment:
Minor: I think the original text wanted to refer to the `album admin page` and the general concept of `artist`, not specifically the classes with that name, so adding code markup would not be helpful.